uploading_packages/artifactory/artifactory_ce.rst

```diff
@@ -3,7 +3,7 @@
 Artifactory Community Edition for C/C++
 =======================================
 
-Artifactory Community Edition (CE) for C/C++ is the recommended server for development and hosting private
+`Artifactory Community Edition (CE) for C/C++ <https://jfrog.com/community/open-source/>`_ is the recommended server for development and hosting private
```

Comment:
This page is a bit confusing, and shows different solutions, not that evident to the newbie what thing needs to be download, plus it also features some things like ``IDE Integrations``, that might lead to confusion, thinking there are Conan integrations with those IDEs? 

Plus that page, what it does is to redirect back to https://conan.io/downloads.html, so maybe point to that one directly? One jump less

Reply:
I put that one a touch lower when it says "you can download it by" but I was trying to add a generic marketing page about the Community Edition but there's not many picks

Maybe just not put anything?

Reply:
```suggestion
Artifactory Community Edition (CE) for C/C++ is the recommended server for development and hosting private
```

Reply:
Yeah, maybe dropping it is the best.

Reply:
👍